cgroup1/state.go

```diff
@@ -14,7 +14,7 @@
    limitations under the License.
 */
 
-package cgroups
+package cgroup1
 
 // State is a type that represents the state of the current cgroup
 type State string
```

Comment:
Does it make sense to share types like State between cgroup1 and cgroup2?

Reply:
Not necessary to share